runtime/manual/basics/modules/index.md

```diff
@@ -94,63 +70,3 @@ All functions, classes, constants and variables which need to be accessible
 inside external modules must be exported. Either by prepending them with the
 `export` keyword or including them in an export statement at the bottom of the
 file.
-
-## FAQ
```

Comment:
I am also of the opinion that FAQs are evidence of bad documentation. Glad this is being removed : )